Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -44,6 +44,7 @@ def select_rows(view, row_indices, command=QItemSelectionModel.ClearAndSelect):
 
 class OWCorrespondenceAnalysis(widget.OWWidget):
     name = "Correspondence Analysis"
+    short_name = "Corresp\nAnalysis"
```

Comment:
Would "Correspondence" fit? It not, I'm OK with this.

Reply:
Correspondence is the word that elides in the original name.